OP-301 Add Dashboard to nav menu

axelpezzo opened the pull request:
The code add the Dashboard link for both the menus (Desktop and Mobile).

almeida1492 commented:
@axelpezzo I think that having a link to the dashboard in the navigation menu at the upper side corner is redundant while in a **desktop context**. Notice that we have a breadcrumb trail in the main portion of the header. A dashboard link is constant to this breadcrumb trail.

The idea of adding it to the menu in a **mobile context** is based on the fact that we don't have the breadcrumb trail in this context.

almeida1492 reviewed (changes requested):
The drawer menu is acting funny. It's showing two buttons with the label "Dashboard" with different sizes. The first one does nothing under a click event though. Another odd thing is the scrolling: it doesn't behave nicely when you try a full scroll down.

axelpezzo commented:
> The drawer menu is acting funny. It's showing two buttons with the label "Dashboard" with different sizes. The first one does nothing under a click event though. Another odd thing is the scrolling: it doesn't behave nicely when you try a full scroll down.

Now, the body scroll is fixed while the mobile menu is open. Regarding the Dashboard link, the smallest it's just the Breadcrumb.
I added this recently in due to a feature request. When the user browses the Dashboard page it might be confusing having the same labels in the menu. I'll take some time to think about how to solve this in the next commit.

almeida1492 commented:
> > The drawer menu is acting funny. It's showing two buttons with the label "Dashboard" with different sizes. The first one does nothing under a click event though. Another odd thing is the scrolling: it doesn't behave nicely when you try a full scroll down.
> 
> Now, the body scroll is fixed while the mobile menu is open. Regarding the Dashboard link, the smallest it's just the Breadcrumb.
> I added this recently in due to a feature request. When the user browses the Dashboard page it might be confusing having the same labels in the menu. I'll take some time to think about how to solve this in the next commit.

Oh... I see. I think we can think of a visual way of showing the user that that's a breadcrumb trail and not a menu button then.

I point out a different bug I've just found here: if you open and close the drawer menu, the whole activity is not scrollable anymore.

axelpezzo commented:
> Oh... I see. I think we can think of a visual way of showing the user that that's a breadcrumb trail and not a menu button then.

Now I think that could be work as expected: I added an Home Icon near the Dashboard breadcrumb's link visible only from Mobile Menu. Let me know if you think that this could work even for the Desktop version.
 
> I point out a different bug I've just found here: if you open and close the drawer menu, the whole activity is not scrollable anymore.

Fixed.
